general/administration/installing.md

```diff
@@ -253,14 +198,6 @@ services:
 > [!Note]
 > For Nvidia Hardware encoding the minimum version of docker-compose needs to be 2. However we recommend sticking with version 2.3 as it has proven to work with nvenc encoding.
 
-### Docker Hub image maintained by LinuxServer.io
-
-<a href="https://hub.docker.com/r/linuxserver/jellyfin"><img alt="Docker Pull Count" src="https://img.shields.io/docker/pulls/linuxserver/jellyfin.svg"></a>
-
-The intent of the Jellyfin image is to provide the capability of building from source using Docker. The native image is [compiled](https://github.com/jellyfin/jellyfin/blob/master/Dockerfile) inside the container whereas the LinuxServer image [installs](https://github.com/linuxserver/docker-jellyfin/blob/master/Dockerfile) Jellyfin as a package. This can lead to larger image size and more space consumption since there isn't a common base image for the official Jellyfin image, [Jellyfin](https://hub.docker.com/r/jellyfin/jellyfin/tags) vs [LinuxServer](https://hub.docker.com/r/linuxserver/jellyfin/tags) sizes. The Jellyfin image does not support environmentals except for $TZ. No other environmental parameter works. If switching between images, the folder locations are not the same, so you have to move config folders around before spinning up the alternative image.
```

Comment:
Again, I added this because there are fundamental differences between the images and people kept asking the 'why pick one over the other' question

Reply:
> A lot of the changes you're gutting were created just a few days ago
> 
> #312
> 
> While I do agree with your assessment, can we find some middle ground? Such as to install docker, please see this `official documentation and guide`

The way I'm thinking about these docs:
- It's out of scope for Jellyfin documentation to describe in detail how to install Docker or to be a tutorial on using Docker.
- Documentation should be factually correct and on point with references to more in-depth resources.
- The previous Docker instructions were quite verbose and rambling. I can tell the author was excited about recently learning about containers and how to use them. Doesn't matter if their contribution is old or new, we should strive to offer the best docs for the community needs.

I think a lot of people want to run JF on Synology with HWA transcoding. We need a guide for that. We might also want a guide like "JF DigitalOcean droplet + GDrive rclone guide" and similar.

Reply:
Hmm, previous comment was regarding "While I do agree with your assessment, can we find some middle ground?"

Reply:
I've made this PR for rclone and mergerfs 😁. I didn't want to over do the guide portion since I linked all the necessary guides. 

https://github.com/jellyfin/jellyfin-docs/pull/325

Reply:
> Again, I added this because there are fundamental differences between the images and people kept asking the 'why pick one over the other' question

We really should squash that question. Why should LinuxServer.io image be supported? I get that they install the i965-va-driver for HWA and some other stuff for raspberrypi. On Fedora I can run HWA on intel with 100% FOSS. It's more that we should obsolesce LinuxServer.io image for Intel and Radeon. Not sure what to do WRT Nvidia.

- LinuxServer.io/jellyfin and jellyfin/jellyfin image sizes are almost identical and I'm not sure which one is claimed to be larger?
- Regarding environment variables: Except timezone what else do people want to change? We should support it if it make sense to make that point moot.

Reply:
I think we might be slightly larger by a few mbs, but I haven't checked in a minute. But honestly, the lsio image is a lot more stable. Our docker nightly is always getting burned one way or another, the nightly is weeks out of date due to build issues. Lsio's is up to date since they install via apt nightly. 

The nvidia env should be included. It wasnt in the Dockerfile until I updated it but never got to the docs.

Reply:
How is the docker image built? I think I can set it up to build using buildah on Github actions or Azuer pipelines. Using qemu or cross compilation for the ARM builds.

Reply:
Currently we use qemu. Build web via tarball, build dotnet, fresh debian, install everything. I like the lsio methodology but I like ours for the ability to modify and dev in a docker environment. 

Afaik, the images are fine, I believe it's the azure pipelines that are broken, but i haven't built since 10.5 nightlies. @dkanada would know more about the azure pipelines if you're interested in helping. I'm off for the night, it was good talking.

Reply:
Well, lsio depends on JF official packages. The only issue I see is the Nvidia stuff, but we should be able to create a image which has everything prepared and only a second layer of proprietary binaries are required for HWA.

I haven't found any files for the container image builds in git.

Reply:
What's the issue for nvidia exactly? I mightve misunderstood your comment. So we did just fix the ffmpeg packages so that they don't require the nonfree Intel packages. What files are you looking for?

Reply:
That NVIDIA might require proprietary drivers.